### PR TITLE
Add rust package tutorial

### PR DESCRIPTION
**Pull request by shuternay**
Add a tutorial how to apply model from rust package.

### CODE REVIEW

File: catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,390 @@
+{
```
Comment (kizill) [inline catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb]: <p>Why did you leave error message in notebook? Moreover, frozendict was never used in this notebook.</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/catboost/catboost/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (kizill) [inline catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb]: <p>We support string class_names, like</p><p>```cb.CatBoostClassifier(class_names=class_names) ```</p>

 _Reply via <a href='https://app.reviewnb.com/catboost/catboost/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (shuternay) [reply catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb]: Fixed

Comment (shuternay) [reply catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb]: Fixed

**Comment by shuternay** [comment]
I fixed issues for both java and rust notebooks

**Comment by kizill** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@kizill, internal review request created: [818267](https://a.yandex-team.ru/review/818267)

**Comment by arcadia-devtools** [comment]
Check [818267](https://a.yandex-team.ru/review/818267) passed

**Pull request merged**